pkgs/applications/networking/mailreaders/himalaya/default.nix

```diff
@@ -5,47 +5,31 @@
 , installShellFiles
 , installShellCompletions ? stdenv.hostPlatform == stdenv.buildPlatform
 , installManPages ? stdenv.hostPlatform == stdenv.buildPlatform
-, pkg-config
-, Security
-, libiconv
-, openssl
 , notmuch
-, withRusttlsTls ? true
-, withRusttlsNativeCerts ? withRusttlsTls
-, withNativeTls ? false
-, withNativeTlsVendored ? withNativeTls
 , withImapBackend ? true
 , withNotmuchBackend ? false
 , withSmtpSender ? true
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "himalaya";
-  version = "0.7.3";
+  version = "0.8.0";
 
   src = fetchFromGitHub {
     owner = "soywod";
     repo = pname;
     rev = "v${version}";
-    sha256 = "HmH4qL70ii8rS8OeUnUxsy9/wMx+f2SBd1AyRqlfKfc=";
+    sha256 = "kK/F3Geiuz0CgGSE0sCOfSZjg9gEPLyUEilzb+SdIM8=";
```

Comment:
Replace `sha256` with `hash`

Reply:
Just by curiosity, what does it change? I see a lot of derivations with `sha256`.

Done :heavy_check_mark: